appserver/tests/application/src/test/java/org/glassfish/main/test/app/appclient/AppClientTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2025 Contributors to the Eclipse Foundation.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package org.glassfish.main.test.app.appclient;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+
+import org.junit.jupiter.api.Test;
+
+import static org.glassfish.main.itest.tools.GlassFishTestEnvironment.getGlassFishDirectory;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.stringContainsInOrder;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ *
+ */
+public class AppClientTest {
+
+    @Test
+    void noArgsAndHelp() throws Exception {
+        File appclient = new File(getGlassFishDirectory(), "bin/appclient");
```

Comment:
Script name should be `bin\appclient.bat` in Windows.

Reply:
Oh, yeah, there is the suffix ...